client/src/components/app.jsx

```diff
@@ -1,18 +1,48 @@
 import React from 'react';
+import $ from 'jquery';
+import ListingList from './ListingList.jsx';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-
+      expanded: false,
+      mockData: {
+        reviews: [],
+      }
     };
+    this.getMockData = this.getMockData.bind(this);
```

Comment:
You don't have to bind the mockData function because you aren't passing it into a listener. You're just invoking it in a componentDidMount

Reply:
I thought about this a bit more, and I changed my mind. If you were to set up a listener later and wanted to use this function, this would be helpful, and it doesn't change how the function works now.